includes/admin/class-wc-subscriptions-admin.php

```diff
@@ -1203,29 +1214,32 @@ public static function has_settings() {
 	 */
 	public static function get_settings() {
 
-		return apply_filters( 'woocommerce_subscription_settings', array(
-
+		return apply_filters(
```

Comment:
```suggestion
		/**
		 * Filter the settings for the Subscriptions extension.
		 *
		 * @param array $settings Array of settings in the format required by the woocommerce_admin_fields() function.
		 */
		return apply_filters( 'woocommerce_subscription_settings', array(
```

Adds a filter comment block.